src/util/Constants.js

```diff
@@ -1019,10 +1019,22 @@ exports.InteractionResponseTypes = createEnum([
  * * BUTTON
  * * SELECT_MENU
  * @typedef {string} MessageComponentType
+ * @deprecated Use @{@link ComponentTypes} instead
```

Comment:
That’s not how you link
```suggestion
 * @deprecated Use {@link ComponentTypes} instead
```